Composer/packages/client/src/components/Header.tsx

```diff
@@ -138,22 +140,29 @@ const calloutDescription = css`
 // -------------------- Header -------------------- //
 
 export const Header = () => {
-  const { setAppUpdateShowing, setLocale } = useRecoilValue(dispatcherState);
+  const { setAppUpdateShowing, setLocale, setWebChatPanelVisibility: toggleWebChatPanel } = useRecoilValue(
```

Comment:
The `dispatcherState` is used twice and `toggleWebChatPanel` has two aliases

Reply:
since this is not a functional problem, I pushed a commit ahead to fix it

Reply:
thanks @yeze322